lib/tapjoy/ldap/key/show.rb

```diff
@@ -0,0 +1,25 @@
+module Tapjoy
+  module LDAP
+    module Key
+      # Show all of a user's keys
+      class Show
+        def show
+          username = opts[:user]
```

Comment:
we should probably do the user validation done other places here to avoid weirdness when we use this value as key in the next line.